docs/pages/docs.mdx

```diff
@@ -21,6 +21,10 @@ Documentation is:
 - Not marketing although it plays a role in onboarding customers. Page views matter, but accuracy and customer satisfaction matter most.
 - Often essential to leadership when they're evaluating a product. Doubly so when potential customers are in high-regulation spaces like finance or government. Many teams neglect their documentation only to fail to secure (or have delayed) the lucrative contracts that such potential customers can bring.
 
+<Admonition type="tip">
+  Read through the great: https://www.divio.com/ documentation to get a sense about the kind of documentation we want to transform our current documentation into.
```

Comment:
Any particular reason for Divio, vs other docs?

Reply:
Was recommended by @klizhentas and I agree it's pretty great - we'll probably include several other exemplars in here in the future as well: https://storybook.js.org/docs/react/get-started/introduction and https://blueprintjs.com/docs/ are some other great one's.